docs/natspec-format.rst

```diff
@@ -0,0 +1,202 @@
+Solidity contracts can have a special form of comments that form the
+basis of the Ethereum Natural Specification Format.
```

Comment:
This sounds like people already know what the Ethereum Natual Specification Format is. I think we should introduce the term here instead and explain what it is meant to be used for.